SignalMessaging/utils/OWSAvatarBuilder.m

```diff
@@ -230,14 +237,34 @@ + (void)drawInitialsInAvatar:(NSString *)initials
     [initials drawAtPoint:drawPoint withAttributes:textAttributes];
```

Comment:
This is probably be fine, but it'd be better not to use `[NSString drawAtPoint: withAttributes:]`.

Reply:
IIRC Apple has provided a number of alternatives over the years.  My memory is fuzzy, but I think there's TextKit, Core Text, and even another way I think.  These newer approaches provide more accurate handling of i18n and subtleties of font rendering.  We're doing measurement now to a) center the text in the avatar b) ensure text fits within the avatars bounds.  I believe that kind of measurement is more accurate with these other approaches.  Anyway, these other approaches are more work IIRC.

NBD since this works and this kind of precision doesn't matter in the avatars.